ibmcloud/cluster/ansible/tasks/k8s.yaml

```diff
@@ -6,12 +6,12 @@
 ---
 - name: "Add gpg key "
   apt_key:
-    url: https://packages.cloud.google.com/apt/doc/apt-key.gpg
+    url: https://pkgs.k8s.io/core:/stable:/v1.28/deb/Release.key
```

Comment:
Sounds like the k8s version is same as kubeadm by default as described in https://kubernetes.io/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm/#kubeadm-s-skew-against-the-kubernetes-version, I'm wondering is it possible to get the version from kubadm --version and set it dynamically?